units/en/conclusion/conclusion.mdx

```diff
@@ -1,5 +1,14 @@
 # Conclusion
 
-This course is in-progress and has not yet concluded!
+Thank you for participating in the Machine Learning for 3D Course!
 
-To receive updates as the course releases, sign up for the course mailing list [here](https://mailchi.mp/911880bcff7d/ml-for-3d-course).
+This has been a high-level overview of what's going on at the intersection of Machine Learning and 3D. For further exploration:
+
+- Keep up with [3D Arena](https://huggingface.co/spaces/dylanebert/3d-arena) for the latest projects.
```

Comment:
Could be cool to add a Hub URL with tag tracking all the open models as well for example: https://huggingface.co/models?pipeline_tag=image-to-3d&sort=trending